src/hooks/useTransactionPreparation.ts

```diff
@@ -68,13 +69,10 @@ export default function useTransactionPreparation({
       // @ts-ignore
       const messageFeeType = sourceApi.registry.createType('MessageFeeData', {
         lane_id: laneId,
-        payload: u8aToHex(payloadType.toU8a())
+        payload: payloadType.toHex()
       });
 
-      const estimatedFeeCall = await sourceApi.rpc.state.call<Codec>(
-        estimatedFeeMethodName,
-        u8aToHex(messageFeeType.toU8a())
-      );
+      const estimatedFeeCall = await sourceApi.rpc.state.call<Codec>(estimatedFeeMethodName, messageFeeType.toHex());
```

Comment:
what's the benefit fo this reformatting? personally , i prefer the two lines

Reply:
🤷 don't ask me, I just obey prettier :)